### ISSUE TITLE
Can't focus math-field using space when using a screen reader

### ISSUE THREAD

**Issue by Leon-Lj** [arnog/mathlive#2298]
When using a screeen reader you can navigate a page using up/down arrow. Normally when you reach a textarea you can focus it by pressing space, but this does not work the math-field. Pressing tab works for focus though.

https://github.com/arnog/mathlive/assets/4263639/17ef713a-ef36-4bb2-abbb-161fc9d95cc3

This problem happens with both NVDA and JAWS.

### Steps to Reproduce

> (Required) Provide steps that are specific and repeatable

1. Start NVDA
2. Start Chrome (chrome should be started after NVDA, sometimes it has problems if it's not)
3. Go to https://cortexjs.io/mathlive/demo/
4. Use down arrow to navigate to the math-field.
5. Press space

### Actual Behavior

Nothing happens, it does not focus in the math-field.

### Expected Behavior

It should focus the math-field, allowing input using the keyboard.

### Environment

**MathLive version** 
0.98.6
0.98.5
0.94.8

**Operating System**
Windows 11

**Browser**
Chrome 121 using NVDA 2023.3.3
Firefox 122 using NVDA 2023.3.3
Chrome using JAWS

**Comment by arnog** [comment]
I have made a change that seems to work with VoiceOver. Unfortunately, I don't have a Windows system handy to test with NVDA, but let me know if this is still an issue.

**Comment by Leon-Lj** [comment]
Unfortunately that did not solve the problem.

The problem was introduced by commit 640abd54c9cbbffb6c7c1f61d2f7daed7707d6c5. If I revert it on todays master, setting `delegatesFocus: false` and `tabindex=-1` then it works fine and I can focus using space in NVDA.

However, I would guess that this breaks Firefox focus again? I tested is briefly in Firefox and it seemed to still work, but I'm guessing the error was more subtle than simply not being able to focus.

**Comment by mgreminger** [comment]
I'm also noticing that setting `delegatesFocus` to `false` causes the `focusin` event to not trigger consistently for math fields in both Firefox and Chrome.

**Comment by mgreminger** [comment]
I installed NVDA and tested with Chrome and, unfortunately, the space bar focusing only works when both `tabindex=-1` and `delegatesFocus`  is `false`.  Thought it could be related to this [NVDA issue](https://github.com/nvaccess/nvda/issues/13026), but your pen shows that it should be possible.

**Comment by Leon-Lj** [comment]
Indeed, it only works in NVDA when both `tabindex=-1` and `delegatesFocus: false` are set.

My understanding is that having `delegatesFocus: true` and `tabindex=0` is the correct way of doing it. That it doesn't work in NVDA (and JAWS) feels like a bug with NVDA. But it's not as simple as it not working with shadowRoot and `delegatesFocus: true` (as shown by the codepen example), so what is special about the usage here that makes it break?
